src/platform-includes/enriching-events/breadcrumbs/automatic-breadcrumbs/apple.mdx

```diff
@@ -1,5 +1,6 @@
 The Cocoa SDK captures breadcrumbs automatically for:
   - Application lifecycle events (`didBecomeActive`, `didEnterBackground`, `viewDidAppear`)
   - Touch events
```

Comment:
I think we should remove `- Touch events`, we create breadcrumbs for UIButton `.tapInside` control event, we dont cover touches in the app.

What do you think @philipphofmann ?

Reply:
Yes, please update this in this PR 👍